bench/micro/Bench/Database/LSMTree/Monoidal.hs

```diff
@@ -67,37 +64,39 @@ benchConfig = Common.defaultTableConfig {
 -- contents of the resulting database are the same.
 benchInsertsVsMupserts :: Benchmark
 benchInsertsVsMupserts =
-    env (pure $ snd $ randomEntriesGrouped 800_000 250) $ \ess -> bgroup "inserts-vs-mupserts" [
-        env (pure $ V.map mkNormalInserts ess) $ \inss -> bench "normal-inserts" $
-          withEmptyNormalTable $ \(_, _, _, _, t) ->
-            V.mapM_ (Normal.inserts t) inss
-      , env (pure $ V.map mkMonoidalInserts ess) $ \inss -> bench "monoidal-inserts" $
-          withEmptyMonoidalTable $ \(_, _, _, _, t) ->
-            V.mapM_ (Monoidal.inserts t) inss
-      , env (pure $ V.map mkMonoidalMupserts ess) $ \mupss -> bench "monoidal-mupserts" $
-          withEmptyMonoidalTable $ \(_, _, _, _, t) ->
-            V.mapM_ (Monoidal.mupserts t) mupss
-      ]
+    env (pure $ snd $ randomEntriesGrouped 800_000 250) $ \ess ->
+      env (pure $ V.map mkInserts ess) $ \inss ->
+        bgroup "inserts-vs-mupserts" [
+          bench "normal-inserts" $
+            withEmptyNormalTable $ \(_, _, _, _, t) ->
+              V.mapM_ (inserts t) inss
+        , bench "monoidal-inserts" $
+            withEmptyMonoidalTable $ \(_, _, _, _, t) ->
+              V.mapM_ (inserts t) inss
+        , bench "monoidal-mupserts" $
+            withEmptyMonoidalTable $ \(_, _, _, _, t) ->
+              V.mapM_ (mupserts t) ess
+        ]
     where
       withEmptyNormalTable =
           perRunEnvWithCleanup
             (do (tmpDir, hfs, hbio) <- mkFiles
-                (s, t) <- mkNormalTable hfs hbio benchConfig
+                (s, t) <- mkTable hfs hbio benchConfig
                 pure (tmpDir, hfs, hbio, s, t)
             )
             (\(tmpDir, hfs, hbio, s, t) -> do
-                cleanupNormalTable (s, t)
+                cleanupTable (s, t)
                 cleanupFiles (tmpDir, hfs, hbio)
             )
 
       withEmptyMonoidalTable =
           perRunEnvWithCleanup
             (do (tmpDir, hfs, hbio) <- mkFiles
-                (s, t) <- mkMonoidalTable hfs hbio benchConfig
+                (s, t) <- mkTable hfs hbio benchConfig
                 pure (tmpDir, hfs, hbio, s, t)
             )
             (\(tmpDir, hfs, hbio, s, t) -> do
-                cleanupMonoidalTable (s, t)
+                cleanupTable (s, t)
                 cleanupFiles (tmpDir, hfs, hbio)
             )
```

Comment:
Also here, there isn't really a distinction anymore, so we can remove one of them